training-material.md

```diff
@@ -13,7 +13,7 @@ The most recent training VM includes:
 
 ### TL;DR
 
-- Training VM system account : `thehive/thehive1234`
+- Training VM system account : `admin/thehive1234`
```

Comment:
Move this line to precede the current line 19 with the reference to Cortex.

Text should read, "TheHive superAdmin account : `admin/thehive1234`

Reply:
there is a difference btw system and application accounts.